components/mpas-ocean/cime_config/config_compsets.xml

```diff
@@ -147,6 +147,11 @@
     <lname>2000_DATM%JRA-1p4-2018_SLND_MPASSI%DIB_MPASO%IBDISMFDATMFORCED_DROF%JRA-1p4-2018_SGLC_SWAV</lname>
   </compset>
 
+  <compset>
+    <alias>GMPAS-JRA1p5-DIB-PISMF-DIS</alias>
```

Comment:
@matthewhoffman, do we need to add a test with this mode?  I assume that's the motivation for changing the years for the `ocn_glcshelf` test but then a new test isn't actually here.  Maybe we want to add something like:
```
ERS_Ld5.TL319_oQU240wLI_ais8to30.GMPAS-JRA1p5-DIB-PISMF-DIS.mpaso-ocn_glcshelf
```
to the end of `e3sm_cryo_developer`:
https://github.com/E3SM-Project/E3SM/blob/eb0b97655ceea7c6d6a7d9b147fdd481dee3f238/cime_config/tests.py#L128

Reply:
I ran:
```
./create_test --wait --walltime 1:00:00 ERS_Ld5.TL319_oQU240wLI_ais8to30.GMPAS-JRA1p5-DIB-PISMF-DIS.chrysalis_intel.mpaso-ocn_glcshelf
```
and it passed.